### ISSUE TITLE
2.0.7.0 Circuit changes need to be re-examined

### ISSUE THREAD

**Issue by finkledoodle** [GTNewHorizons/GT-New-Horizons-Modpack#4246]
#### Which modpack version are you using?
2.0.7.0

As of 7.0, wetware circuits and bioware circuits were changed in a way that doesn't follow the logic that all previous circuit tiers follow.

In 5.1, Wetware circuit boards require an HV pump, HV circuit, and HV sensor. A bit pricey for a bare circuit board, but all easily automatable, and understandable with the tier 
![unknown1](https://user-images.githubusercontent.com/12854704/54170160-d2f78080-4432-11e9-942e-2fc4f16cfe2f.png)

Same for bio circuits in 5.1 They continue the same logic, using an EV pump, EV circuit, and EV sensor. All still totally automatable without requiring an assembly line. And while expensive, still understandable for a BASIC CIRCUIT BOARD.
![unknown2](https://user-images.githubusercontent.com/12854704/54170168-da1e8e80-4432-11e9-9fc4-3ee2af6069f4.png)

Now in 7.0, things have changed dramatically. Wetware circuitboards now require LuV pumps, LuV sensors, IV circuit, and need LuV eu/t.
![2019-03-11_18 37 12](https://user-images.githubusercontent.com/12854704/54169331-7fcffe80-442f-11e9-9a00-a72424acb10e.png)

Bio circuit boards now require UV pumps, UV sensors, and ZPM circuit, and require UV! eu/t to make!
![2019-03-11_18 37 15](https://user-images.githubusercontent.com/12854704/54170205-fae6e400-4432-11e9-82a0-0404b30763c3.png)

Furthermore, while the recipe for bio medium has not changed since 5.1 and is still completely broke in terms of how expensive and time-consuming it is, the recipe for growth medium has been significally nerfed in a way that makes zero sense. 

As of 5.1 the recipe for raw growth medium called for mince meat, tiny pile of salt, sugar, and distilled water. This recipe makes sense in terms of GTNH's desire for a balance between realism and balance. As of 7.0, the 3 choices players have for growth medium all involve extremely expensive options that serve only to deter the player from making growth medium as much as they can. Americium dust, cosmic neutronium, ichorium nuggets, pearl fragments, etc. These are not recipes that are appropriate for raw growth medium, nor do they make more sense for the recipe than the original.

From my perspective, a bacterial growth medium being made from meat, sugar, salt, and water seems much more realistic and fitting than some magical cosmic neutronium or other assorted elements that seem to have replaced it solely for the reason of making it more of a grind.

#### What do you suggest instead/what changes do you propose?

Until now, the general logic for circuits has been that each base version of a circuit tier nanoprocessor, quantumprocessor, etc. has been a cheaper and easier to make way of making that tier of circuit. IE: It is easier to make a quantumprocessor assembly than a nano supercomputer, or a wetwareprocessor than a crystalprocessor assembly. However with this update, the amount of effort requiring in creating even the base wetware or bio circuit makes crafting higher tiers of crystalprocessors look much easier in comparison, breaking that entire chain of logic that has held for some time.

Revert the changes for Wetware and bio circuit boards to requiring HV and EV components, respectively.  Revert the change for raw growth medium to use a more realistic recipe rather than calling for magic (ichor/pearl fragment), fusion-created (americium), or space (cosmic neutronium) components. Alternatively, make use of GT++'s new manure/poo related materials for a more realistic solution for bacterial-based components.

Furthermore, as bio medium calls for mutagen in large quantities, adding a recipe to allow mutagen to be extracted from cavernous veins would be both a renewable and balanced source, especially as bees have been nerfed in 7.0

**Comment by voidlogic** [comment]
+1 for cutting the unrealistic growth medium recipes. 

ambivalent about the circuit boards. we do get 16 per in new recipe.

**Comment by finkledoodle** [comment]
One of the issues that I've heard and said myself about the circuit board change is that not every player knows how to automate the assembly line, and with how circuits are needed in extreme quantities at LuV and above for large projects, that could prove extremely troublesome for many players.

**Comment by 0lafe** [comment]
On the topic of the things like bio medium, growth medium, etc we have a long ass bio setup concept from @spluff that I think we're working off. Bart has already added some of the machines for it, and it looks like a bunch of fun. It gives most things renewable recipes iirc, mostly at the cost of power and some difficulty in setup. Idk how much has been added, but I know there's a LOT to it. 

For LuV parts, it's a tier below the tier you can make the circuit in. Idk why bios use UV, they probably should be using ZPM and zpm voltages. Two LuV parts for 16 circuits is 1 LuV part per mainframe, which isn't that bad. Considering the mainframe is a UHV circuit, that seems kinda ok. Having it be 1 HV instead seems weirdly low. By the end of zpm, a single LuV sensor shouldn't be that bad of a craft if you're trying to make a wetware mainframe, or even 16 LuV circuits. By the end of UV for bios, one zpm sensor is fine per 16 zpms or 2 UEVs. If the uptiered versions get added, then using 1 UV sensor for 64 LuV circuits seems not bad by UHV. 

Maybe we change it from a sensor to something that doesn't use circuits though? That doesn't seem great. Pump + piston maybe?

**Comment by finkledoodle** [comment]
I understand the change from being 1 to 1 to being 16 to 16 for the circuit boards, and the tradeoff from being 16 HV parts to 1 LuV part. I guess one of my main reservations is just that a lot of people don't know how to automate the assembly line if it is going to be required in such a large scale (and circuits will require it to be so). I'd be willing to meet in a middle ground if there was a quest that helped players automate their assembly lines so that folks like me can understand. 

As for bio/growth medium, when you mention how you have a "long ass bio setup concept" that isn;t appealing. I've already got a dozen other machine setups running constantly that I have to worry about. What is wrong with the existing setup of salt, water, mean, sugar, that has to change so much? Furthermore you're implementing these changes without fixing the big issue of bio circuitry being totally broken, which is much more of a priority to players like me.

I don't need things made more difficult for arbitrary reasons at this tech stage where it takes tens of thousands of resources to accomplish anything. While I understand your desire to make each tier of circuitry cost Tier-1 of materials, that is a completely arbitrary rule that has no reason to be held to some gold standard. For example: yes the wetware circuit board uses an HV pump, HV circuit, and HV sensor to make. So what? While it doesn't follow some rule of using IV parts to make an LuV circuit, its still annoying to do.

When you involve even higher tier circuits, such as base bio circuits, it becomes even more clear. The recipe for a bio circuit board (which is needed just to make ONE base bioprocessor) in 7.0 calls for a UV pump, UV sensor, and ZPM circuit. Now, even if this was downgraded as olafe suggested to a ZPM pump and ZPM sensor, those are both annoying and costly things to make. And this is just to make a circuit board!

**Comment by AlexBucket** [comment]
I think parts need to be reverted to what they are in 5.1 and something better come up with. If it were me, I'd revert the raw bio medium and ask for input into a better option that the current space/magic route that makes it more about BIO and CHEMISTRY kinda thing.
Yes, meat/salt/sugar/water is a bit easy for that tier and by all means change it, but make it more about biochemistry than just 'this is a high tier material, lets use that'.

I would be reverting the whole bio circuits and similar thing, ask for input. I have doubts there is more than a handful of regular players who are at that tier, and i have even more doubts they're wondering 'maybe these circuits should be harder'.

**Comment by draknyte1** [comment]
We now have Poo/Raw Animal Waste to use in BioChem.
/2cents

**Comment by Xauronx** [comment]
Fron finkle:

> "Until now, the general logic for circuits has been that each base version of a circuit tier nanoprocessor, quantumprocessor, etc. has been a cheaper and easier to make way of making that tier of circuit. IE: It is easier to make a quantumprocessor assembly than a nano supercomputer, or a wetwareprocessor than a crystalprocessor assembly. However with this update, the amount of effort requiring in creating even the base wetware or bio circuit makes crafting higher tiers of crystalprocessors look much easier in comparison, breaking that entire chain of logic that has held for some" 

I want to add, that the lower-Tier Wetwares or Biocircuits are straight up useless, if making an LuV Crystal Circuit is much cheaper in comparison. Why keep this way to make them in steps in the first place, if the only viable Circuit you want is the highest Tier one in that Line? In general it feels just like throwing extra steps into your way to drag things out.

On this note - Vote for Animal Poo flowing through our bases!

**Comment by 0lafe** [comment]
The animal poo is cool, and I think spluff was looking for a "waste" type product for something, but the poo stuff pales in comparison to the bio things coming. I really wish I had any way to show you what it contains, but it's amazing. 

As for wetwares being worse than 2 crystals, I think that comes down to a problem in the circuit production idea of the new circuit being the better option. First off, it doesn't have to be the best option right away. Nanos are great, but if you don't have good epoxy in EV, they might not be the right choice for you. Similarly with wetwares, even using 1 LuV part per 8 LuV circuits, it's not too bad. Maybe not early UV, but by the time you've automated production of LuV parts and wetwares, I wanna say they'll beat making the chips for crystals. LuV parts are 2 tiers below the main tier of wetware production, it's not really that huge of a cost. Using HV is just weird with it being 5 tiers below. In large amounts, needing the 2x SoCs kind of sucks, and with a growth medium/stemcell change, I can see wetwares being competitive options to crystals. 

However the other side of this is a weird circuit system. Each tier you go up, you essentially get 4x speeds of the tier before you. Once you've made it 4 tiers up it becomes more of a 2x but 4x is still possible. Granted this is a little weirder with the ++ multis, but considering they're faster than the speeds I'm taking about, I'll leave them out of this. With materials being less of an issue, and processing times being the main block to making things in these tiers, the speed rule is about all that matters in circuit creation. 1 crystals processor become 4 crystal processors if you tier up, so to fit the idea of 2 crystals being worth 1 wetware in the next tier, wetwares need to be craftable with a 2x speed compared to crystals in the tier before. 1 crystal in LuV, is 4 in zpm, which is equivilant to 2 wetwares. If you have the ability to make 2 wetwares in ZPM for the cost of making 1 crystal in LuV, then by UV you can make 8 wetwares for the cost of 4 bios. This progression of circuits needs either a very very hard cap on the ability to make the circuits to keep the cost of a circuit the same in each tier, otherwise they get progressively easier by 2x for each tier after. This ends up with a weird circuit progression of them getting easier each tier, which doesn't really make all that much sense. 

I think the biggest problem with the circuits is the ability to make them into the higher tiers. If that stopped with quantum's in IV, then crystals were a single step UV circuit, wetwares a single step UHV, etc, it might create a better lategame circuit system? I'm not too sure if that's what we want, it does mean you're only using LuV+ circuits when you need a UV+, and you're using all of the tiers of quantums ideally, which seems a little weird. Idk what to do about it to be honest, but I think we might wanna look at the system as a whole. Another option is to add SoC versions of more of the quantum line maybe? Could use higher tier PiCs and stuff to make say a quantum assembly in 1 step with zpm stuff, a UV version of the supercomputer and UHV mainframe? This would probably push the normal quantum SoC to LuV though? Otherwise zpm has the base circuit and assembly, or it would push the whole thing from zpm to the mainframe single-step being UEV. This frees up the LuV+ circuits to be independent of IV- scaling and let them actually get harder each tier instead of easier.

**Comment by Xauronx** [comment]
Finkle quote:

> As for bio/growth medium, when you mention how you have a "long ass bio setup concept" **that isn;t appealing**. I've already got a dozen other machine setups running constantly that I have to worry about. What is wrong with the existing setup of salt, water, meat, sugar, that has to change so much? 
**Furthermore you're implementing these changes without fixing the big issue of bio circuitry being totally broken**, which is much more of a priority to players like me.

0lafe quote:

> The animal poo is cool, and I think spluff was looking for a "waste" type product for something, but the poo stuff pales in comparison to the bio things coming. I really wish I had any way to show you what it contains, but it's amazing.

Please fix things first before adding huge new game changers! Seriously, this would prevent a lot of discussions like this! <.<

Finkle quote:

> I understand the change from being 1 to 1 to being 16 to 16 for the circuit boards, and the tradeoff from being 16 HV parts to 1 LuV part. **I guess one of my main reservations is just that a lot of people don't know how to automate the assembly line if it is going to be required in such a large scale (and circuits will require it to be so).** I'd be willing to meet in a middle ground if there was a quest that helped players automate their assembly lines so that folks like me can understand.

0lafe quote:

As for wetwares being worse than 2 crystals, I think that comes down to a problem in the circuit production idea of the new circuit being the better option. First off, it doesn't have to be the best option right away. Nanos are great, but if you don't have good epoxy in EV, they might not be the right choice for you. Similarly with wetwares, even using 1 LuV part per 8 LuV circuits, it's not too bad. **Maybe not early UV, but by the time you've _automated_ production of LuV parts and wetwares**, I wanna say they'll beat making the chips for crystals. LuV parts are 2 tiers below the main tier of wetware production, it's not really that huge of a cost. Using HV is just weird with it being 5 tiers below. In large amounts, needing the 2x SoCs kind of sucks, and with a growth medium/stemcell change, I can see wetwares being competitive options to crystals.

What do you think about that topic brought up 0lafe? Please consider this point brought up.

Finkle quote:

> **I don't need things made more difficult for arbitrary reasons at this tech stage where it takes tens of thousands of resources to accomplish anything.** 

0lafe quote:

> With materials being less of an issue, and processing times being the main block to making things in these tiers, the speed rule is about all that matters in circuit creation.

Honestly two very different experiences clash here. Clearly some other players see the "all that matters in circuit creation" differently. The main Question here being: Does speed matter to the general player base? Or just to _you_ who has already beaten the pack and know all the nicks to it? Only a handful of people reach this tier anyway why give the player base just more and more insentive to give up at some point? You dont even get acess to much cool stuff in this tier, let it be some new toys in magic, new mods opening up or cool multiblock contraptions. So it just fells like an **arbitary** grind in that stage without any meat to it. 

In general I feel you are just defending your nerfs without really adressing many of the points brought up by finkle :(

**Comment by 0lafe** [comment]
>Honestly two very different experiences clash here. Clearly some other players see the "all that matters in circuit creation" differently. The main Question here being: Does speed matter to the general player base? Or just to you who has already beaten the pack and know all the nicks to it? Only a handful of people reach this tier anyway why give the player base just more and more insentive to give up at some point? You dont even get acess to much cool stuff in this tier, let it be some new toys in magic, new mods opening up or cool multiblock contraptions. So it just fells like an arbitary grind in that stage without any meat to it.
In general I feel you are just defending your nerfs without really adressing many of the points brought up by finkle :(

So the power vs materials thing, he's right that they take tons of materials. However in these tiers you're given tools to make all these materials near infinite, and at least exceed your usage rate. This results in a system in which processing of these materials into their usable circuit parts is the only input needed, beyond the power input of the creation of the resources. It's partly why I have a problem with current seaweed, because the only way to get it without manual harvesting is with robets. A lategame crystal circuit takes 1 NbTi, 1 YBaCuO, 8 platinum, some sulfuric, some copper, 288L epoxy, 2 carbon, and a crystal chip which needs 1 emerald, 1B helium, and about 16L europium (more like 17-18 if you're doing the recycling method of creation). None of this is difficult to come by in UV, most ores you can have over 100k of without much difficulty, which covers most of the costs. 16L europium is a fusion thing now, but only takes 1.6s in a mk1 reactor, and only .4s in a mk3 which is the UV version. Epoxy is just oil automation, maybe with manual phosphor input, but even using ores for phosphor you can still end up with insane amounts of epoxy. The main costs in the circuit are just the PLE energy/time cost for the chip. Everything here is easily automated by the tiers in question, and can give outputs that exceed your use by a good amount. If you struggle to make crystal chips you're missing part of the automation process, it's not the circuit being too expensive. Maybe you have manual fusion reactors, or super slow ore processing, but when adjusted for the speeds you can achieve using actual power from these tiers, you shouldn't be having a problem with it.

Wetwares cost 1 YBC, a little silicone/aluminum/poly, 32 carbon, some glowstone, a wafer, the lower tier of SoC used in the crystal, 250L uu, 1000L coolant, 500L growth medium, 4 HSS-S, 8 styrene (should be *anyRubber), 2 fluxed electrum, 4 PBI, 4 reinforced glass, 16 stemcells, 8 NbTi, like 16L PTFE, 4 platinum, some sulfuric, 288L epoxy, 1/16 of an IV circuit (can be read like 1/16 of the crystal recipe above), an LuV sensor every 16 crafts, and an LuV pump for the same amount. Again, all of these "materials" are really really easy to get in bulk in these tiers. Your bottleneck will really just be uu production, and the power you put into it. The issue currently is growth medium isn't the best recipe, and the creation of stemcells is also a little subpar. UU however can be made at a reasonable rate if you're willing to dump some power into it, and especially if we want to account for the GT++ uu gen. That said, I'm also in favor of adding lategame sources for uu, either through fusion or possibly with a new multi @spartek has made recently. 

If you would really like, I can go through and write out speeds for each option here and give you a general idea of how long it'll take to make a crystal/wetware at zpm or UV, but idk if it is really needed. Everything oil based can be automated pretty easily to give you tons and tons of it, way more than crafting these circuits eat. Quarries are LuV, and pretty cheaply so in 2.0.7, giving you the ore materials needed here as most have ores by this point. Osmium, iridium, tungsten, niobium, titanium, FE components, neodynium, etc. 

Point is, yes you dump materials in, however these aren't really materials in the sense of being limited or needing you to do much to get them. They're really just additional power sinks, giving the end of it a sort of easy to calculate speed at a tier, which is what I'm saying would essentially need to increase 2x with wetwares, and 4x with bios to make them accommodate being better use than crystals. With the bio changes, these become even more apparent, while also increasing what you need to do to make all of these resources simply power driven. It's super doable in these tiers, and I'd argue intended, but maybe with bios we can lower some times of wetwares to give you more of an output? That said, I wouldn't single out the LuV parts as the slowdown, and without the current growth medium/stemcell recipes, I feel as if wetwares might actually be better than crystals once you have the setup for them. 

Bios are another thing, but it's the same idea. Even the bio medium is mostly just a power sink at the end of the day. Infinity is purely power, MC too, oriharukon doesn't even really take power considering its just used as a dust and comes from a pure ore, SpNt is just a lower infinity cost but still just power. The low tier circuit parts aren't even close to a power sink as the speeds you use them in are attainable in like IV or LuV. While the power costs are different, the idea is still the same. Given an amount of power you are suggested to spend, you will get a certain speed of circuit production out of it. 

>What do you think about that topic brought up 0lafe? Please consider this point brought up.

There should be a quest saying it's doable and you should do it. Maybe hint at helpful mods (OC/SFM?), but I think that covers it. It's kind of intentionally a bit of a puzzle that you're meant to work out, but it's not a bad idea to point people towards it being a puzzle to begin with. 

>Please fix things first before adding huge new game changers! Seriously, this would prevent a lot of discussions like this! <.<

Thing is this "huge new game changer" is the fix. It's kind of a long system so implementing it might take more than a version, which is a bit of a sad time, but it's coming. That said, the only use requiring bios/post bio (nano line) is single block UEVs, stargates, and solars ZPM+. The current creation method isn't exactly broken as much as it's just not that great. It's all doable stuff, it's just weirdly expensive in some areas (looking at seewead). Think of them more as place holders until the new bio recipes come. It might have been a mistake adding new circuits before we have the foundation to craft those circuits, but uh, it happened. Big endgame problems are not really having many materials to use, especially for niche things like bio mutagen and the like. The current bio medium recipe is probably a result of that issue, but I'm not too sure how it came to pass tbh (oriharukon dust in the same recipe as infinity? like, what?). Bios aren't really a go to for replacing wetwares yet as a singlestage circuit, but they are craftable and semi fit their required uses.

**Comment by 0lafe** [comment]
Oh all that said, I totally support changing the sensor to something else. Maybe a piston, maybe something else, but right now it seems a little bad considering it takes 4 LuV circuits itself lol. The IV circuit in the recipe is pretty much just symbolic at that point when the other part takes essentially 8x of them

**Comment by AlexBucket** [comment]
> However in these tiers you're given tools to make all these materials near infinite, and at least exceed your usage rate. This results in a system in which processing of these materials into their usable circuit parts is the only input needed, beyond the power input of the creation of the resources.

Is this merely an assumption? As a UV player, I still find myself regularly running out of materials. I have access to the Deep Dark, I have an ender quarry, I use my blood pick and still find myself lacking materials. Time gates and material gates are still very much a thing.

Just a bit of a background for Finkle, Auron and I:
We are 3 ZPM - UHV (ish) players on ProsperCraft, playing seperate of each other. I think all 3 of us have similar issues regarding time and material gates. Are all 3 of us playing wrong? Well, we have 3 quite different methods of play, and yet we still come to the same issue. Mind you, we're not complaining about being material gated, but you seem to completely disregard materials being relevant because you may not of had that issue.
We also play on a server that regularly has 20+ players during peak times. We all have to be considerate of more than just us playing on the server? TPS become a real issue. Sure materials can be infinate. We could all run world accelerated apiaries or multiple ender quarries, but that comes at the risk of the server being unplayable for everyone.

> Thing is this "huge new game changer" is the fix. It's kind of a long system so implementing it might take more than a version, which is a bit of a sad time, but it's coming

This sounds promising. By why are your cards to tightly held to your chest? This straight up affects 3/4 players commenting here, and the only one it won't affect straight away is the one with all the info? Has it not been mentioned time and time again recently something about commmunity input?
Don't go dropping huge changes from left field. Give us the info, let us help make things balanced, as we're REGULAR players, playing at the tier that it'll affect. There can't be more than a handful of regular players getting about, let alone ones that comment on the official discord or git. Let us help mate.

**Comment by Xauronx** [comment]
0lafe quote:

> So the power vs materials thing, he's right that they take tons of materials. **However in these tiers you're given tools to make all these materials near infinite, and at least exceed your usage rate.** This results in a system in which processing of these materials into their usable circuit parts is the only input needed, beyond the power input of the creation of the resources. **It's partly why I have a problem with current seaweed, because the only way to get it without manual harvesting is with robets.** A lategame crystal circuit takes 1 NbTi, 1 YBaCuO, 8 platinum, some sulfuric, some copper, 288L epoxy, 2 carbon, and a crystal chip which needs 1 emerald, 1B helium, and about 16L europium (more like 17-18 if you're doing the recycling method of creation). None of this is difficult to come by in UV, most ores you can have over 100k of without much difficulty, which covers most of the costs. 16L europium is a fusion thing now, but only takes 1.6s in a mk1 reactor, and only .4s in a mk3 which is the UV version. Epoxy is just oil automation, maybe with manual phosphor input, but even using ores for phosphor you can still end up with insane amounts of epoxy. The main costs in the circuit are just the PLE energy/time cost for the chip. Everything here is easily automated by the tiers in question, and can give outputs that exceed your use by a good amount. If you struggle to make crystal chips you're missing part of the automation process, it's not the circuit being too expensive. Maybe you have manual fusion reactors, or super slow ore processing, but when adjusted for the speeds you can achieve using actual power from these tiers, you shouldn't be having a problem with it.

So we are talking about the need to "constantly" run every possible rescource generator to keep up with higher tier demand? Quarry every planet + DD and Overworld Dimensions? If so goodbye to server TPS. Sweet. And we are still not adressing the arbitary grind of making things hard for the sake of making it hard. I like to quote a famous game company representive here: "If it's not fun, why bother?" And fix seaweed before kicking up recipies.

0lafe quote:

> Thing is this "huge new game changer" is the fix. It's kind of a long system so implementing it might take more than a version, which is a bit of a sad time, but it's coming. That said, the only use requiring bios/post bio (nano line) is single block UEVs, stargates, and solars ZPM+. The current creation method isn't exactly broken as much as it's just not that great. It's all doable stuff, it's just weirdly expensive in some areas (looking at seewead). Think of them more as place holders until the new bio recipes come. **It might have been a mistake adding new circuits before we have the foundation to craft those circuits, but uh, it happened.**

Exactly!!! my point. My problem with the current nerf is, that costs were adjusted **_before_** a fix was set into place. This is just bad game design in my opinion that shouldn't happen. And ->

Finke quote:

> **As for bio/growth medium, when you mention how you have a "long ass bio setup concept" that isn;t appealing.** I've already got a dozen other machine setups running constantly that I have to worry about.

If you let those circuits costs sit in addition to packing a huge new bio line next patch, i will say it again: "If it's not fun, why bother?" Hard to see it in a good light at this point when things get hammered before inteded systems are set in stone. 

0lafe quote:

> There should be a quest saying it's doable and you should do it. Maybe hint at helpful mods (OC/SFM?), but I think that covers it. It's kind of intentionally a bit of a puzzle that you're meant to work out, but it's not a bad idea to point people towards it being a puzzle to begin with.

It is in its entirely a big puzzle alone too figure out an entire mod, to automate one multiblock, not a **bit**. Again, making things hard for the sake of it behing hard. Give more tools to players, but not lay everything out for them i get that. But this part is requiered/crucial for progressing in this game, why just not give anything here? Give them at least the means to understand, what is requiered here! 

0lafe quote:

> Bios are another thing, but it's the same idea. **Even the bio medium is mostly just a power sink at the end of the day. Infinity is purely power, MC too, oriharukon doesn't even really take power considering its just used as a dust and comes from a pure ore, SpNt is just a lower infinity cost but still just power.** The low tier circuit parts aren't even close to a power sink as the speeds you use them in are attainable in like IV or LuV. While the power costs are different, the idea is still the same. Given an amount of power you are suggested to spend, you will get a certain speed of circuit production out of it.

Yeah essentially everything can only be or seem like a power sink at some point if there is no content left to work with :D Doesnt make it fun tho. Why flesh out the grind in endgame when there is not enough of it?

**Comment by 0lafe** [comment]
>So we are talking about the need to "constantly" run every possible rescource generator to keep up with higher tier demand? Quarry every planet + DD and Overworld Dimensions? If so goodbye to server TPS. Sweet. And we are still not adressing the arbitary grind of making things hard for the sake of making it hard. I like to quote a famous game company representive here: "If it's not fun, why bother?" And fix seaweed before kicking up recipies.

Nothing like that. Constant e quarry in like 1-2 dims seems fine. Constant being while online, not 24/7. Bees and stuff help a ton too. They gotta be fixed for some things, but crops I think work on everything they should. Also, what do you mean by "hard"? If you mean stupidly long times even at tier voltages, then yeah that doesn't seem great. If you mean setting up more automation and infrastructure to progress, isn't that kind of the point of the game? 

>If you let those circuits costs sit in addition to packing a huge new bio line next patch, i will say it again: "If it's not fun, why bother?" Hard to see it in a good light at this point when things get hammered before inteded systems are set in stone

If you don't like long convoluted fluid processing with logistically challenges then what's the point of oil, or even fusion in some ways. The only real cost in these circuits right now that aren't great are ones that change completely with bios. No one's saying the circuits should use less yttrium, it's less see weed per circuit, or a better recipe for growth medium. All of those are being addressed with these changes, so the costs of them are really being lowered in a sense. 

>It is in its entirely a big puzzle alone too figure out an entire mod, to automate one multiblock, not a bit. Again, making things hard for the sake of it behing hard. Give more tools to players, but not lay everything out for them i get that. But this part is requiered/crucial for progressing in this game, why just not give anything here? Give them at least the means to understand, what is requiered here!

Except that's kind of what it is now. You'll notice that manually running the assline is too slow, and there's a quest saying it can be automated. It takes some understanding of stuff, but it's doable and not even that bad. If you can't do it, or don't want to for whatever reason, ask someone who's done it how. Or ask for help. However, everything in the game already points you towards it and gives you what you need. 

>Yeah essentially everything can only be or seem like a power sink at some point if there is no content left to work with :D Doesnt make it fun tho. Why flesh out the grind in endgame when there is not enough of it?

What's the grind though? If you mean you currently make bio medium too slow, it's not like it has a use really, and the rate of production kind of fits the usage of it. If you want another circuit line past wetwares that's viable, then it's being worked on. But considering it doesn't even really have a use, I'm not sure what's so bad about it currently. It's changing yes, and one day it'll have uses, but right now most of the stuff is just placeholding until there's a better recipe. Specifically the bio stuff, we're waiting on Bart/spluff for. 

There's a difference between a grind, and requiring infrastructure of that tier. It's not really a grind to make crystalprocessors in ZPM, in the same way it won't be for wetwares in UV (post bio change), or bios in UHV. While creating these items before those tiers might be more difficult, I wouldn't call it grindy. 

It's not about adding grind, it's about adding tier appropriate challenges to them. In many cases that's more of an EU/t requirement, but if you have any better ideas I'd love to hear them. Although considering you don't think the assline is a fair challenge to automate, then idk if we agree on this. There's things we can do like ask for high tier thaum setups with auto infusions, the new bio stuff having some weird mechanics for production, but there's not too much that lategame struggles in doing. The problem is most things that even take substation setups end up using power to run, which is the only cost for that. Idk a way around that tbh. As long as we have power positive systems with large outputs, and ways to turn that power into materials, it kind of becomes all you can really do. While using materials without power infinite solutions, you end up needing to mine them which isn't much better, I'd even say it's worse.

**Comment by voidlogic** [comment]
> Nothing like that. Constant e quarry in like 1-2 dims seems fine. Constant being while online, not 24/7. Bees and stuff help a ton too. They gotta be fixed for some things, but crops I think work on everything they should. 

Using enderquarry for ore products past UHV-tier is laughable.

Even if I ran 16!! ender quarries 24/7 in deep dark it still wouldn't even account for 10% of the material that I yeet into wetwares every week on version 2.0.5.1.

Concerns about increasing material quantity costs from where they already stood in version 2.0.5.1 are completely valid, and I think 'grind' is a fair word to describe such a predicament. 

I look forward to the processing chain that is in the works for new bio medium though! It sounds exciting!

**Comment by 0lafe** [comment]
What takes that many materials in a wetware though? Or what takes that many wetwares? I could see you doing 10k kinda easily, thats like less than 100k of most things, which is doable DD numbers. If you're on prospercraft not having speed 3 is kind of a problem, but even so. Especially on 5.1 with the costs being even lower, what is there to drive the costs up that high? 10k wetwares using <100k of I think all the ingredients, that's 10k LuV circuits or 2.5k mainframes. Unless you're making a Stargate, which I only say because I have no idea what most of the recipes are, then what would you even be using that main wetwares for? Even solars, you'd be making more than you need with that amount of circuits. That's 2.5k UHV sensors/emitters of circuits, which I think is even more than a Stargate would use. Not to mention these are material numbers that are well below what you can really do with quarries assuming you're allowed more than 1/dimension, or even just put them in multiple dimensions. With crops, that's really just a 4x to what you're getting, which is kind of huge. If you pick planets based on what ores you need too, I wanna say it'll cover your costs easily.

**Comment by voidlogic** [comment]
>Especially on 5.1 with the costs being even lower, what is there to drive the costs up that high? 10k wetwares using <100k of I think all the ingredients,

Asserting that the material costs are fine without having ever actually made a wetware yourself.

Nice.

**Comment by 0lafe** [comment]
I've made thousands?... But you still didn't answer my question. How many are you making, what's your bottleneck to making more? No way a quarry can only give you 10% unless you're making an absurd amount of them (like more than 10k), or you're doing something else really, really wrong.

**Comment by voidlogic** [comment]
If you've ever actually made a wetware, then you would have easily realized that your projected material cost is ORDERS OF MAGNITUDE incorrect.

See my AE below projecting cost for 100 wetwares (I have enough floating stock to keep the numbers clean for that amount). A single deep dark quarry would take months to mine this amount of resource.

![1](https://user-images.githubusercontent.com/10283881/54305144-13730d80-459d-11e9-8eb8-e277c42cf541.JPG)
![2](https://user-images.githubusercontent.com/10283881/54305156-15d56780-459d-11e9-8c85-629e1a7ddf33.JPG)
![3](https://user-images.githubusercontent.com/10283881/54305159-179f2b00-459d-11e9-963f-e1b220c18f35.JPG)
![4](https://user-images.githubusercontent.com/10283881/54305163-1b32b200-459d-11e9-8ac4-7007834b3edb.JPG)
![5](https://user-images.githubusercontent.com/10283881/54305168-1cfc7580-459d-11e9-8033-31bcfd4e944c.JPG)
![6](https://user-images.githubusercontent.com/10283881/54305172-1ec63900-459d-11e9-8686-478d04b9824a.JPG)
![7](https://user-images.githubusercontent.com/10283881/54305178-21289300-459d-11e9-9c35-24a6cabe9a4a.JPG)

**Comment by boubou19** [comment]
then use both quarrieS, bees and crops silmutaneously.

**Comment by voidlogic** [comment]
I'm done with this thread.

**Comment by Xauronx** [comment]
I am too. We are talking against walls <.<

**Comment by 0lafe** [comment]
Yes, mainframes, not single circuits. You can see the cost difference in the board cost. 1600 for 100 circuits, so each mainframe on base takes 16x the cost of a wetware, considering the real cost is in the board. You could make about 800 wetware circuits for that cost, minus the stuff that goes to the 64 SC per mainframe, of which you're making 50 stacks. There's also some stuff there that comes from the excessive use of HV over LuV parts. Like the stainless, the gold, and easily a crazy silver cost. The gold on just HV motors for the boards is 12800, and the silver is even higher. A mainframe will take 4 zpm pumps to make, but also 1 LuV pump. How is the LuV pump the problem lol. 

The high stuff there that's not changing is the Ti and niobium, which are kinda high. Maybe we want to lower it a little, or use a more tier appropriate item? It's a latezpm/early UV circuit, could use americium foil maybe? Like 2x ingots/wetware? would mean 3200 for 100 mainframes, which seems kinda fair for a UHV circuit, as well as being a little better than 14400 NbTi. Also, HSS-G is used a little less with the LuV rotor being HSS-S, which dilutes some of those costs a little for some more osmium/iridium. The zpm rotor is also now naq alloy, but maybe we wanna reduce how many are in the circuit? 64x ZPM SC is a little bit of a weird cost for the mainframe, what about 16x UV? Puts the cost at 1 UV pump per mainframe, which seems a little better. 

Do keep in mind this is a UHV circuit though, its kind of meant to have a lot go into it, but the more we can remove the dumb parts of the recipe that dump weird amounts of badly tiered items (nanos, HV parts, etc) and replace them with better amounts of correctly tiered parts, the better these recipes get. No reason a UHV circuit should just be a dump for an EV dust, and an IV dust, when it can be a good amount of a zpm+ material 

@voidlogic get yer ass back here and suggest a better way to reduce the straight material cost to something that's actually tier appropriate

**Comment by 0lafe** [comment]
Also wanna point out a single quarried vein gives 3.2k ish ore of each, which is 12.8k with Combs/bees, and drilling it gives like 20k+ before Combs, which turns to 80k+. With the copper/gold/iron coming mostly from other sources, even just quarrying gives you like 100 UHV circuits from getting a few veins. Bees/crops are also not terrible lategame with Bart's machines making crop Gene shit not really an issue, and gendustry for bees doing the same (considering it needs almost top tier bees to make, the mutation aspect is almost less useful). There's also WAs to make breeding easier, and crafting frames for bees shouldn't really be hard considering the power you have/materials. 

There's also the part where these are UHV circuits, which like, even 100 is kind of a lot of.

**Comment by Prometheus0000** [comment]
This is old, if you have a problem with them given the changes since, make a new ticket, also see #6531.
